### PR TITLE
Article page code review

### PR DESCRIPTION
**Pull request by El-Potato-Slayer**
In this project we applied multiple use cases of grid, flexbox and one use case of float on the aside tag to recreate the New York Times article page.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
- Kindly replace the ```ids``` with ```classes```. Classes are used for styling

Thank you for the review. There is one thing I disagree with. Both IDs and classes are used for styling. IDs are meant for styling that will only apply to one element and classes are reusable; [see here](https://css-tricks.com/the-difference-between-id-and-class/). Unless I'm writing a CSS library, could you please provide me with a valid reason to use classes all the time instead of IDs?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+        <a class="smallest" href="">Education Rate</a>
+        <hr />
+        <a class="smallest" href="">Mobile Applications</a>
+        <a class="smallest" href="">Replica Edition</a>
+        <a class="smallest" href="">Español</a>
+        <a class="smallest" href="">中文网</a>
+      </div>
+    </div>
+    <hr />
+    <div id="footer-bottom" class="flex smallest" style="justify-content: space-between; padding: 0 56px;">
```
Comment (andresporras3423) [inline index.html]: Don't use inline styles

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+}
+
+footer div div a:hover {
+  text-decoration: underline;
+}
+
+#suggestion-section a,
+#most-popular h3 a {
+  text-decoration: none;
+}
```
Comment (andresporras3423) [inline styles.css]: This selector has the same style definition of selection on line 25. Group selectors with same style definition.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+#suggestion-section {
+  width: 1200px;
+  grid-template-columns: 3fr 1fr;
+  grid-gap: 24px;
+  margin-bottom: 100px;
+}
+
+footer div div a:hover {
+  text-decoration: underline;
+}
```
Comment (andresporras3423) [inline styles.css]: This selector has the same style definition of selection on line 21. Group selectors with the same style definition.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+  border: 1px solid #333;
+}
+
+#suggestion-section hr {
+  border: none;
+  height: 1px;
+  background-color: #909090;
+}
+
+#comment-button {
```
Comment (andresporras3423) [inline styles.css:179]: Don't overuse ID selectors

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @El-Potato-Slayer @nikoescobal** 
Good job with your new  submission for this project :muscle:  
the page really looks like a clone of the original :clap: :clap: :clap:  
Your project passes most of the requirements. There are just a few details to improve before you can go ahead:
-  [ ] Don't overuse ID Selectors.
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).
- [ ]  Don't use inline styles in your HTML file.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: styles.css (hunk 4 above)
Comment (El-Potato-Slayer) [reply styles.css:179]: Thank you for the review. Could you please provide a value for the amount of IDs that need to be used? As we mentioned to the previous TSE, we'd love to understand why IDs are an issue for styling.

**Review by trekab — APPROVED** [review]
# Approved! :tada:
Hey @team, well done! :clap: You have met the project requirements. The reason it is not a good practice to overuse `id` selectors is that it is not possible to share those styles among your elements. This reduces any opportunities you have to refactor your CSS styles.

__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**